### DISCUSSION TITLE
Is there an equivalent to npm_package_ variables?

### DISCUSSION THREAD

**Discussion by davesteinberg** [denoland/deno#26399] [Q&A]
I'm wondering if Deno provides an equivalent to npm's [package.json vars](https://docs.npmjs.com/cli/v10/using-npm/scripts#packagejson-vars). It would be handy to reference, say, the package name or version defined in deno.json from the tasks in that file and the scripts they run (via Deno.env). I couldn't find anything documented, but I thought I'd ask, just in case.

**Comment by marvinhagemeister** [comment]
Related issue https://github.com/denoland/deno/issues/25342
